Marlin/ubl_G29.cpp

```diff
@@ -566,7 +583,7 @@
     // When we are fully debugged, the EEPROM dump command will get deleted also. But
     // right now, it is good to have the extra information. Soon... we prune this.
     //
-    if (code_seen('J')) g29_eeprom_dump();   // EEPROM Dump
+    if (code_seen('j')) g29_eeprom_dump();   // EEPROM Dump
```

Comment:
I don't think we'll be adding lower-case parameter values. GCode is supposed to be case-insensitive.